lib/fizzbuzz.rb

```diff
@@ -11,3 +19,6 @@ def run(input)
 #You don't necessarily need to execute this script to complete this challenge, but how would you "run" this method (pun intended) so that it printed a value to the terminal?
 #
 #HINT: it's an instance method.
+# - Return "Fizz" if the number is divisible by 3
+# - Return "Buzz" if the number is divisible by 5
+# - Return "FizzBuzz" if the number is divisible by 3 & 5
```

Comment:
I think what Jay was looking for here was something like `SuperFizzBuzz.new.run(5)`.